code/__HELPERS/game.dm

```diff
@@ -122,16 +122,27 @@
 	for(var/client/remove_from in show_to)
 		remove_from.images -= image_to_remove
 
+
 ///Add an image to a list of clients and calls a proc to remove it after a duration
-/proc/flick_overlay(image/image_to_show, list/show_to, duration)
+/proc/flick_overlay_global(image/image_to_show, list/show_to, duration)
 	for(var/client/add_to in show_to)
 		add_to.images += image_to_show
 	addtimer(CALLBACK(GLOBAL_PROC, GLOBAL_PROC_REF(remove_images_from_clients), image_to_show, show_to), duration, TIMER_CLIENT_TIME)
 
-///wrapper for flick_overlay(), flicks to everyone who can see the target atom
-/proc/flick_overlay_view(image/image_to_show, atom/target, duration)
+/// Atom-wrapper for flick_overlay_global that converts an iconstate
```

Comment:
This doesn't describe what it actually does, which is more important than its implementation.

Reply:
This wasn't resolved.